posthog/hogql_queries/web_analytics/test/test_web_stats_table.py

```diff
@@ -1006,50 +1006,67 @@ def test_cohort_test_filters(self):
         assert results == [["/path1", 1, 1]]
 
     def test_timezone_filter(self):
-        d1, s1 = "d1", str(uuid7("2024-07-30"))
-        d2, s2 = "d2", str(uuid7("2024-07-30"))
+        date = "2024-07-30"
```

Comment:
For me to be *suuuper* confidant this works, I'd like to see some tests that cover before / after the daylight savings change for the same timezone

Reply:
Just added it, good call